src/components/ColumnEntry.tsx

```diff
@@ -32,6 +39,12 @@ export const ColumnEntry = ({ column, uuid, onChange, onDelete }: Props) => {
 		onDelete(uuid);
 	}, [onDelete, uuid]);
 
+	const handleKeyPress = (e: React.KeyboardEvent<HTMLInputElement>) => {
+		if (e.key === 'Enter') {
+			onEnter();
```

Comment:
```suggestion
			e.preventDefault();
			onEnter();
```

Don't forget to prevent the default behavior whenever you're adding functionality that goes against the norm; just in case someone forgets in the future about this and wraps the `<input />`s in a `<form>` e.g.